cpp/oneapi/dal/algo/basic_statistics/backend/basic_statistics_interop.hpp

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+* Copyright 2020-2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include "oneapi/dal/algo/basic_statistics/common.hpp"
+#include "oneapi/dal/backend/interop/common_dpc.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+
+#include <daal/include/algorithms/moments/low_order_moments_types.h>
+
+namespace oneapi::dal::basic_statistics::backend {
+
+namespace daal_lom = daal::algorithms::low_order_moments;
+namespace interop = dal::backend::interop;
+
+using task_t = task::compute;
+using descriptor_t = detail::descriptor_base<task_t>;
+
+inline auto get_daal_estimates_to_compute(const descriptor_t& desc) {
+    const auto res_op = desc.get_result_options();
+    const auto res_min_max = result_options::min | result_options::max;
+    const auto res_mean_varc = result_options::mean | result_options::variance;
+
+    if ((res_op.test(res_min_max) && res_op.get_mask() ^ res_min_max.get_mask()) ||
+        (res_op.test(res_mean_varc) && res_op.get_mask() ^ res_mean_varc.get_mask())) {
+        return daal_lom::estimatesAll;
+    }
+    else if (res_op.test(res_min_max)) {
+        return daal_lom::estimatesMinMax;
+    }
+    else if (res_op.test(res_mean_varc)) {
+        return daal_lom::estimatesMeanVariance;
+    }
```

Comment:
Can we simplify such conditions?

Reply:
simplified, via adding ~ operator for result_option_id